packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -114,13 +128,35 @@ function accumulateDispatches(
   }
 }
 
+function accumulateCapturedDispatch(inst, phase, event) {
+  if (inst && event && event.dispatchConfig.captured) {
```

Comment:
Oh interesting, what I was thinking was something like:

```
if (inst && event && event.dispatchConfig.registrationName) {
  const captured = `${event.dispatchConfig.registrationName}Captured`;
  
    // ...
}
```

This would automatically add it for all dispatch events so you don't have to migrate all the view configs. This is actually how it works on Web where you can use a capture phase handler for _any_ event by adding `Capture` to the end: https://reactjs.org/docs/events.html#supported-events.

Ideally, we would remove the different `captured` and `bubbled` properties from bubbling events as well, and just hard code the `Captured` the same way (almost all, or maybe all, are the same already).

Reply:
Good idea! Have taken the suggestion, only using suffix `Capture` to align with how `phasedRegistrationNames` does it